ydb/core/tx/schemeshard/schemeshard_schema.h

```diff
@@ -1316,8 +1316,8 @@ struct Schema : NIceDb::Schema {
 
         struct MaxRetries : Column<27, NScheme::NTypeIds::Uint32> {};
 
-        struct RowsBilled : Column<28, NScheme::NTypeIds::Uint64> {};
-        struct BytesBilled : Column<29, NScheme::NTypeIds::Uint64> {};
+        struct UploadRowsBilled : Column<28, NScheme::NTypeIds::Uint64> {};
+        struct UploadBytesBilled : Column<29, NScheme::NTypeIds::Uint64> {};
```

Comment:
this won't work: you can't rename columns

Reply:
Fixed